### PR TITLE
Fix: Check PropertyPath value for add error to form

### PR DESCRIPTION
**Pull request by Coosos**
| Q               | A
| --------------- | -----
| Branch?         | >= 1.4
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | yes
| Deprecations?   | no
| Related tickets | https://github.com/Sylius/Sylius/issues/9745#issuecomment-485510417
| License         | MIT

Hello,

When adding a new validation for add product to cart, I realized that "getPropertyPath" was empty at which this causes a 500 error

sylius_inventory_500

```json
{
    "code": 500,
    "message": "Child \"\" does not exist."
}
```

And it is also the case for "InStockValidator", if one provokes the violation one also has this error.

An issue also proposes this way of correcting
    - https://github.com/Sylius/Sylius/issues/9745#issuecomment-485510417

I do not know if it's the best idea to do it like that, but after when we look in Symfony class it is an empty chain which is the default value.
https://github.com/symfony/validator/blob/master/Context/ExecutionContext.php#L85

Tell me your opinion :)

sylius_inventory_400

Bye

### CODE REVIEW

**Comment by Coosos** [comment]
Sorry @lchrusciel I have clicked on repeat request not on purpose

File: src/Sylius/Bundle/OrderBundle/Controller/OrderItemController.php [hunk 1]
```diff
@@ -227,7 +227,11 @@ private function getCartItemErrors(OrderItemInterface $orderItem): ConstraintVio
     private function getAddToCartFormWithErrors(ConstraintViolationListInterface $errors, FormInterface $form): FormInterface
     {
         foreach ($errors as $error) {
-            $form->get('cartItem')->get($error->getPropertyPath())->addError(new FormError($error->getMessage()));
+            $formSelected = (is_null($error->getPropertyPath()) || empty($error->getPropertyPath()))
```
[the remark this answers is not in the document]
Comment (Coosos) [reply src/Sylius/Bundle/OrderBundle/Controller/OrderItemController.php]: Yes, this second condition is not necessary ;)

**Comment by Coosos** [comment]
Question, branch 1.4 is still active in bug fixing or not ? For rebase to fix CI

**Comment by lchrusciel** [comment]
1.4 is not maintained anymore, let's target 1.5

**Comment by lchrusciel** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "1.4"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by Coosos** [comment]
@lchrusciel I just execute command of your message and then pull + fix conflict and push. Now I have 6 commits, what can I do to fix this? Thanks

**Comment by lchrusciel** [comment]
I've created separate PR with these commits (so you are still an author ;)). Hope, you don't mind. Thanks for your contribution.

Closing in favor of #11018

**Comment by Coosos** [comment]
@lchrusciel 
No problem, thank you for your help ;)

**Pull request closed without merging**